tests/tiglFuselageSegment.cpp

```diff
@@ -712,3 +712,55 @@ TEST_F(TiglFuselageSegmentSimple, getSectionCenter)
     ASSERT_EQ(TIGL_MATH_ERROR, tiglFuselageGetSectionCenter(tiglHandle, "segmentD150_Fuselage_1Segment2ID", eta_out_of_range, &pointX, &pointY, &pointZ));
 }
 
+TEST_F(TiglFuselageSegmentSimple, getSectionArea)
+{
+    // test first fuselage segment
+    double eta = 0.;
+
+    double area = 0.;
+
+    double pi = 3.14159265359;
```

Comment:
There is actually a M_PI macro in <cmath> which defined Pi. Please use this instead.